deploy/api.yaml

```diff
@@ -0,0 +1,188 @@
+openapi: "3.0.1"
+info:
+  title: "Experian KBV Wrapper Api"
+  version: "1.0"
+x-amazon-apigateway-request-validators:
+  body:
```

Comment:
can we wire in the `body` validator please? See https://github.com/alphagov/di-ipv-cri-fraud-api/blob/a2ca801bff1dc636ef8ad9df95f33bc20d31a28f/deploy/api.yaml#L33